woocommerce/woocommerce-ios
# CoreDataManager: PSC Recovery Mechanism

### Details:
This PR implements a recovery mechanism in the `loadPersistentStores` callback. Analog to [WPiOS's failsafe.](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Utility/ContextManager.m#L261).

Closes #142

cc @mindgraffiti @bummytime 

### Testing:
- Verify the app builds correctly.
- Verify the code makes sense!!.
- Verif y a Fresh Install / Relaunch works as expected.
